tests/by-util/test_cp.rs

```diff
@@ -497,6 +497,38 @@ fn test_cp_strip_trailing_slashes() {
     assert_eq!(at.read(TEST_HELLO_WORLD_DEST), "Hello, World!\n");
 }
 
+#[test]
+fn test_cp_parents() {
+    let (at, mut ucmd) = at_and_ucmd!();
+
+    let result = ucmd
+        .arg("--parents")
+        .arg(TEST_COPY_FROM_FOLDER_FILE)
+        .arg(TEST_COPY_TO_FOLDER)
+        .run();
+
```

Comment:
please run rustfmt on the test too

Reply:
Oops, sorry about that. I didn't realize `cargo fmt` didn't pick it up. Done. Thanks!

Reply:
Oh, no worries :)